products/bash/release-mq.sh

```diff
@@ -63,6 +63,37 @@ done
 echo "INFO: Tracing support currently disabled"
 tracing_enabled=false
 
+echo "INFO: Setting up certs for MQ TLS"
+QM_KEY=$(cat $CURRENT_DIR/mq/createcerts/server.key | base64 -w0)
+QM_CERT=$(cat $CURRENT_DIR/mq/createcerts/server.crt | base64 -w0)
+APP_CERT=$(cat $CURRENT_DIR/mq/createcerts/application.crt | base64 -w0)
+
+cat << EOF | oc apply -f -
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: mtlsmqsc
+  namespace: $namespace
+data:
+  example.ini: |-
+    Service:
+      Name=AuthorizationService
+      EntryPoints=14
+      SecurityPolicy=User
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: mqcert
+  namespace: $namespace
+data:
+  tls.key: $QM_KEY
+  tls.crt: $QM_CERT
+  app.crt: $APP_CERT
+type: Opaque
+EOF
+
```

Comment:
Only needed if using the custom image, maybe move down into the `else` part of `if [ -z $image_name ]; then` ?

Reply:
done